From 1e14adf9c34e168e28ac43242ec7998c214c6420 Mon Sep 17 00:00:00 2001
From: Filipp Marulin <alifeisfine@gmail.com>
Date: Wed, 27 May 2020 23:46:37 +0300
Subject: [PATCH] disable firmware copy

---
 pre-build.sh | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/pre-build.sh b/pre-build.sh
index 30518e6..9e2ba40 100755
--- a/pre-build.sh
+++ b/pre-build.sh
@@ -44,6 +44,3 @@ find scheduler -name '*.c' -exec grep EXPORT_SYMBOL {} + \
     | awk -F'[()]' '{print "#define "$2" amd"$2" //"$0}'\
     >> include/rename_symbol.h
 
-FW_DIR="/lib/firmware/$KERNELVER"
-mkdir -p $FW_DIR
-cp -ar /usr/src/amdgpu-18.50-756341/firmware/amdgpu $FW_DIR
-- 
2.25.1

